src/EventHub/EventHub.sln

```diff
@@ -18,6 +18,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ScenarioTest.ResourceManage
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestFx", "..\..\tools\TestFx\TestFx.csproj", "{BC80A1D0-FFA4-43D9-AA74-799F5CB54B58}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "KeyVault", "..\KeyVault\KeyVault\KeyVault.csproj", "{7A4B0770-E6C4-479F-9EF4-6C549F217B3C}"
```

Comment:
Please don't reference other module in module project. In most situation reference them in test project is enough.

Reply:
the reference is EventHub.sln, similar to how its done on Compute